zc_common/remote_resource/metadata.py

```diff
@@ -0,0 +1,28 @@
+from django.db.models.fields import related
+from rest_framework.utils.field_mapping import ClassLookupDict
+from rest_framework_json_api.metadata import JSONAPIMetadata
+
+from zc_common.remote_resource.models import GenericRemoteForeignKey
+from zc_common.remote_resource.relations import RemoteResourceField
+
+
+class RelationshipMetadata(JSONAPIMetadata):
+    relation_type_lookup = ClassLookupDict({
+        related.ManyToManyDescriptor: 'ManyToMany',
+        related.ReverseManyToOneDescriptor: 'OneToMany',
+        related.ForwardManyToOneDescriptor: 'ManyToOne',
+        RemoteResourceField: 'ManyToOne',
+        GenericRemoteForeignKey: 'ManyToOne'
+    })
+
+    def get_field_info(self, field):
+        field_info = super(RelationshipMetadata, self).get_field_info(field)
+        if isinstance(field, RemoteResourceField):
+            model_class = getattr(field.parent.Meta, 'model')
```

Comment:
Add error handling, the serializer (`field.parent`) might not have a `model` specified in its Meta